src/common/containers/landing.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import { Anchor } from '../components/button';
+
+import styles from './container.css';
+
+class Landing extends Component {
+  componentWillMount() {
+    if (this.props.auth.authenticated) {
```

Comment:
We shouldn't be causing users to rely on the landing page for documentation.  IMO the way to get back to the landing page is to sign out.